response_operations_ui/controllers/sample_controllers.py

```diff
@@ -32,10 +32,7 @@ def get_sample_summary(sample_summary_id):
 def upload_sample(short_name, period, file):
     logger.info('Uploading sample', short_name=short_name, period=period, filename=file.filename)
 
-    if app.config["SAMPLE_FILE_UPLOADER_URL_ENABLED"]:
-        url = f'{app.config["SAMPLE_FILE_UPLOADER_URL"]}/samples/fileupload'
-    else:
-        url = f'{app.config["SAMPLE_URL"]}/samples/B/fileupload'
+    url = f'{app.config["SAMPLE_URL"]}/samples/B/fileupload'
```

Comment:
this needs to be
 url = f'{app.config["SAMPLE_FILE_UPLOADER_URL"]}/samples/fileupload'